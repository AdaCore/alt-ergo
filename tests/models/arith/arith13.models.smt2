(set-option :produce-models true)
(set-logic ALL)
(declare-const x Int)
(assert (<= x 10))
(push 1)
(maximize x)
(check-sat)
(get-model)
(get-objectives)
(check-sat)
(pop 1)
