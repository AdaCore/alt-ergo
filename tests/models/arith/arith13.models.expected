
unknown
(
  (define-fun x () Int 10)
)
(objectives 
  (x 10))
unknown
